src/components/Cards/QuestionCard/QuestionCard.css

```diff
@@ -0,0 +1,119 @@
+:root {
+	--COLOR-BS-OPTION: #065d8814;
+	--COLOR-CORRECT: #7fffd4;
+	--COLOR-WRONG: #453326;
+	--COLOR-BG-WRONG: #ed8181;
+}
+
+.question__card {
+	color: var(--COLOR-TXT);
```

Comment:
fallback value for colors should be given

Reply:
Will look into it